--- sound/soc/sunxi/sunxi-codec.c.orig	2013-11-29 11:33:58.144224179 +0200
+++ sound/soc/sunxi/sunxi-codec.c	2013-11-29 11:34:04.320224309 +0200
@@ -514,7 +514,7 @@
 	CODEC_SINGLE("Right Capture Mute",SUNXI_ADC_ACTL,31,1,0),
 	CODEC_SINGLE("Left Capture Mute",SUNXI_ADC_ACTL,30,1,0),
 	CODEC_SINGLE("Capture Volume",SUNXI_ADC_ACTL,20,7,0),//录音音量
-	CODEC_SINGLE("Line Volume",SUNXI_ADC_ACTL,13,7,0),
+	CODEC_SINGLE("Line Capture Volume",SUNXI_ADC_ACTL,13,7,0),
 	CODEC_SINGLE("MicL Volume",SUNXI_ADC_ACTL,25,3,0),//mic左音量
 	CODEC_SINGLE("MicR Volume",SUNXI_ADC_ACTL,23,3,0),//mic右音量
 	CODEC_SINGLE("Mic2 Boost",SUNXI_ADC_ACTL,29,1,0),
